roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Install Filebeat.
+  apt:
+    name: filebeat
+    state: present
+
+- name: Create Filebeat configuration directory
+  file:
+    path: "/etc/filebeat/conf.d"
+    state: directory
+
+- name: Copy Filebeat configuration.
+  template:
+    src: filebeat.yml.j2
+    dest: "/etc/filebeat/filebeat.yml"
+    owner: root
+    group: root
+    mode: 0644
+  notify: restart filebeat
+
+- name: Copy Filebeat prospectors
+  template:
+    src: prospector.yml.j2
```

Comment:
Nit: We don't really have .j2 on our templates. Also, we keep them in templates/ with similar path that they'll be deployed to.